src/main/java/org/spongepowered/api/entity/ai/task/AITaskTypes.java

```diff
@@ -38,32 +53,291 @@
     // SORTFIELDS:ON
 
     /**
-     * {@link AITask} where {@link Creature}s avoid other {@link Agent}s based on a {@link Predicate}.
+     * {@link AITask} where {@link Tameable}s attack a {@link Living} which its
+     * {@link Tamer} attacks.
+     */
+    public static final AITaskType ATTACK_AFTER_OWNER = DummyObjectProvider.createFor(AITaskType.class, "ATTACK_AFTER_OWNER");
```

Comment:
Might be nice to create a `private` method in this class just to reduce "boilerplate":
```java
private static AITaskType createDummyFor(String fieldName) {
    return DummyObjectProvider.createFor(AITaskType.class, fieldName);
}
```